fullstack/index.ts

```diff
@@ -1,6 +1,5 @@
 import '@ulixee/commons/SourceMapSupport';
 import {
-  Observable,
   ConnectionToRemoteCoreServer,
   ConnectionToCore,
   InteractionCommand,
```

Comment:
Lint error a few lines down. On purpose? We should add the eslint hint if so

Reply:
Fixed, but it's not showing in my Github UI.